Home about pages

Diana-Duya opened the pull request:
In this PR:

- I created a new repository and cloned it to my local computer.
- Next I set up the linters and created the HTML, CSS and JavaScript files
- I also added the necessary images and icons to the images folder.
- Next, I modified the html, CSS and JavaScript files to create, style the website and also to add features like the mobile-menu.
- The speaker section is  created dynamically in the script.js file.
- I committed the changes after running and fixing linter errors.

Here is a link to the [loom video](https://www.loom.com/share/fa8de27d51b7455097af82a0f88ef136) where i explain more about this project

Diana-Duya committed:
Set up linters and tools

Diana-Duya committed:
initial commit

Diana-Duya committed:
Add chanes to website

Diana-Duya committed:
fix main-menu

Diana-Duya committed:
fix linter errors

Diana-Duya committed:
Edit README

Diana-Duya committed:
Update README.md

Diana-Duya committed:
Update README.md

Diana-Duya committed:
Update README.md

Diana-Duya committed:
Add changes

Diana-Duya committed:
Add changes

Diana-Duya committed:
Fix linters

Diana-Duya committed:
Fix images

Diana-Duya committed:
Fix image

Diana-Duya committed:
Fix image

Diana-Duya committed:
Fix images

Diana-Duya committed:
Fix images

Diana-Duya committed:
fix background

Diana-Duya committed:
fix background

Diana-Duya committed:
fix background

Diana-Duya committed:
fix background

Diana-Duya committed:
fix background

Diana-Duya committed:
fix background

m05e5 commented on index.html:
```diff
@@ -0,0 +1,111 @@
+                <span class="bar bar2"></span>
+                <span class="bar bar3"></span>
+            </div>
+            <div class="main-menu">               
+                    <a class="nav-item nav-active" href="about.html">About</a>
+                     <a class="nav-item" href="">Program</a>
+                    <a  class="nav-item"  href="">Join</a>
+                    <a  class="nav-item" href="">Sponsor</a>
+                   <a  class="nav-item nav-items" href="">Sign in</a>
+            </div>
```
- [ ] Hi. Great job, but you are required to have the logo of your app on the left-hand side of your nav. Kindly add it following the design provided
<img width="973" alt="image" src="https://user-images.githubusercontent.com/37901469/192102180-05c6b744-f94f-4a8e-9cf3-06712b4466bf.png">

m05e5 commented on CSS/style.css:
```diff
@@ -0,0 +1,574 @@
+  padding-left: 20px;
+}
+
+.main-menu {
+  position: absolute;
+  width: 96%;
+  height: 500px;
+  display: flex;
```
- [ ] Please kindly readjust the width of your nav links. on the provided design the nav links take only the right part of the white nav bar

m05e5 commented on CSS/style.css:
```diff
@@ -0,0 +1,574 @@
+  align-items: center;
+}
+
+.footer {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  font-family: 'lato', sans-serif;
```
- [ ] Hi you are required to stick to the design as much as possible so kindly give your footer a `flex-direction: row;` with some space between the logo and the text as it is on the provided design

**your design**
<img width="1436" alt="image" src="https://user-images.githubusercontent.com/37901469/192102680-ca5cb195-3288-4396-84bd-84446f3f924a.png">

**DTF(Design to follow)**
<img width="983" alt="image" src="https://user-images.githubusercontent.com/37901469/192102733-b9901d96-2871-4d28-b4b9-eed899ecb107.png">

m05e5 commented on about.html:
```diff
@@ -0,0 +1,101 @@
+          </div>
+        </div>
+       <div class="two-sections">
+     <section class="partners">
+        <div class="line">
+          <h3 class="partners1">Partners</h3>
+          <hor class="hor2"></hor>
+        </div>
+        <ul class="list">
+            <li><img class="li" src="./img/mozilla.png" alt=""></li>
+            <li><img class="li kakao" src="./img/kakao.png" alt=""></li>
+            <li><img class="li" src="./img/naver.png" alt=""></li>
+        </ul>
+     </section>
+     <footer class="footer">
+     <div class="end">
+        <img class= end-img src="./img/GDG.png" alt="">
+        <h3 class="camp">Google Dev Group.org</h3>
+     </div>
+     <p class="copyright">©2022</p>
+     </footer>
+    </div>
+</section>
```
- [ ] Please you are required to have only the logo and a small note in the about page footer, not partners please kindly fix it 

<img width="1425" alt="image" src="https://user-images.githubusercontent.com/37901469/192102935-af87e424-88cb-4d47-a602-65b1b1c759db.png">

m05e5 commented on README.md:
```diff
@@ -1,72 +1,60 @@
 
 Give a ⭐️ if you like this project!
 
 ## Acknowledgments
```
- [ ] Hi you are required to give appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) in your Acknowledgments section, as required by the [Creative Commons License](https://creativecommons.org/licenses/).

m05e5 reviewed (changes requested):
Hi @Diana-Duya , 

Good job so far. I really like what you did!
There are some issues that you still need to work on to go to the next project but you are almost there!
![mario_ladders](https://user-images.githubusercontent.com/20563252/121369265-4e418880-c901-11eb-848c-1eff06d5a4cf.gif)



## Required Changes ♻️
* You really did a great job but there are some features you need to improve

- [ ] _Please check the comments under this review_

_I'm sure you can do it just keep it up_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_@m05e5_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Diana-Duya committed:
Update README.md

Diana-Duya committed:
Modify css and html

Diana-Duya committed:
Merge branch 'Home-about-pages' of https://github.com/Diana-Duya/Capstone-Project into Home-about-pages

Diana-Duya commented:
Hello @m05e5 , 
Thank you for the review. I have made all the required changes.
- Added a logo
- Added the necessary acknowledgements
- Made adjustments to the footer

mgmediaweb reviewed (approved):
Hi @Diana-Duya, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

![approved6](https://user-images.githubusercontent.com/41912912/192108910-7ba53968-e17b-44bc-b00b-4b49a328775f.gif)

## Highlights 💯 
- Good job following the advice of the previous reviewer ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
